cmd/feed-ingress/main.go

```diff
@@ -14,22 +14,28 @@ import (
 )
 
 var (
-	apiServer  string
-	caCertFile string
-	tokenFile  string
-	debug      bool
+	nginxConfDir string
+	nginxPort    int
+	apiServer    string
+	caCertFile   string
+	tokenFile    string
+	debug        bool
 )
 
 func init() {
 	const (
-		defaultAPIServer  = "https://kubernetes:443"
-		defaultCaCertFile = "/run/secrets/kubernetes.io/serviceaccount/ca.crt"
-		defaultTokenFile  = "/run/secrets/kubernetes.io/serviceaccount/token"
+		defaultAPIServer    = "https://kubernetes:443"
+		defaultCaCertFile   = "/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+		defaultTokenFile    = "/run/secrets/kubernetes.io/serviceaccount/token"
+		defaultNginxConfDir = "."
+		defaultNginxPort    = 80
 	)
 
 	flag.StringVar(&apiServer, "apiserver", defaultAPIServer, "Kubernetes API server URL")
 	flag.StringVar(&caCertFile, "cacertfile", defaultCaCertFile, "file containing kubernetes ca certificate")
 	flag.StringVar(&tokenFile, "tokenfile", defaultTokenFile, "file containing kubernetes client authentication token")
+	flag.StringVar(&nginxConfDir, "nginx-conf-dir", defaultNginxConfDir, "directory to store nginx conf")
+	flag.IntVar(&nginxPort, "nginx-port", defaultNginxPort, "nginx port")
```

Comment:
Think it needs better description, such as `port to serve ingress traffic on`. It's not nginx specific either, in case we have different lbs, so I'd rename the flag to `ingress-port` or something like that.